tutorial/index.md

```diff
@@ -254,9 +254,157 @@ SELECT unnest(pgroonga_snippet_html(
 
 See [`pgroonga_snippet_html` function](../reference/functions/pgroonga-snippet-html.html) for more details.
 
-### Synonym {#synonym}
+## Synonym {#synonym}
```

Comment:
We should not change this.
Synonym is one of full text search topics.

Reply:
@kou Thank you for your comments.
I've fixed them.